src/static/js/pluginfw/plugins.js

```diff
@@ -143,15 +143,144 @@ async function loadPlugin(packages, plugin_name, plugins, parts) {
   }
 }
 
+var matcher = function (name, list) {
+  let matches = [];
+
+  try {
+    const nameRegExp = RegExp(name);
+
+    list.forEach(function (item) {
+      if (nameRegExp.test(item)) {
+        matches.push(item);
+      }
+    });
+  } catch(e) {
+    console.log('ERROR', e);
+  }
+
+  return matches;
+}
+
+function conflictMatcher (parts, name, type) {
+  var type;
+  var opposite = 'pre';
+  var results = [];
+
+  if (type === 'pre') {
+    opposite = 'post';
+  }
+
+  _.each(parts[name][type] || [], function (item_name) {
+    var conflictMatches = matcher(item_name, parts[name][opposite] || []);
+
+    if (conflictMatches.length) {
+      var matchObj = {};
+
+      matchObj[item_name] = conflictMatches;
+      results.push(matchObj);
+    }
+  });
+
+  return results;
+}
+
+function checkPluginConflicts(parts) {
+  var conflicts = [];
+
+  _.chain(parts).keys().forEach(function (name) {
+    var conflictObj = {};
+    var conflictsPre = conflictMatcher(parts, name, 'pre');
+    var conflictsPost = conflictMatcher(parts, name, 'post');
+
+    conflictObj[name] = {pre: conflictsPre, post: conflictsPost};
+    conflicts.push(conflictObj);
+  });
+
+  return conflicts;
+}
+
+function closedChainChecker (parts, name, match, matchCount, res) {
+  var noChain = true;
+  for (var i = 0; i < res.length; i++ ) {
+      var item = res[i];
+
+      if (item[0] === name || item[1] === name) {
+        if (item[0] === match || item[1] === match) {
+          //value allready exists
+          noChain = false;
+            // check which plugins has stricter pre/post definition
+          _.each(parts[match].pre || [], function (item_name) {
+            var matches = matcher(item_name, _.chain(parts).keys().value());
+            if (matches.length > matchCount) {
+              res.splice(i, 1);
+              noChain = true;
+            }
+          });
+          _.each(parts[match].post || [], function (item_name) {
+            var matches = matcher(item_name, _.chain(parts).keys().value());
+            if (matches.length > matchCount) {
+              res.splice(i, 1);
+              noChain = true;
+            }
+          });
+
+          i = res.length;
+          break;
```

Comment:
`i = res.length` is executed immediately before a `break`, thus has no effect.

It may be a logic error, or it can simply be a leftover.